drake/doc/code_style_guide.rst

```diff
@@ -47,12 +47,22 @@ There are several tools for helping you to compile with the required style. They
 ClangFormat
 ^^^^^^^^^^^
 
-(*Notes comming soon*)
+The following instructions were tested on OS X but should also work on Linux. A different installation procedure may need to be followed on Windows.
```

Comment:
Ah, yes.  Drake gives both brew and macports advice in from_source.rst, and I'd forgotten which one was preferred.  I see now that brew is preferred -- so yup, agreed on using brew.